Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -864,9 +724,6 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 		u32 pteg_addr = ((hash & PowerPC::ppcState.pagetable_hashmask) << 6) | PowerPC::ppcState.pagetable_base;
 
-		if ((pteg_addr >> 28) == 1)
-			base_mem = Memory::m_pEXRAM;
-
 		for (int i = 0; i < 8; i++)
 		{
 			u32 pte = bswap(*(u32*)&base_mem[pteg_addr]);
```

Comment:
I actually have a patch locally to do precisely that, but I didn't apply it when I couldn't measure a speedup (due to so few loads taking that path). But I should probably check again just to be sure.